### DISCUSSION TITLE
update sdfat library 2.0.6 to 2.1.2

### DISCUSSION THREAD

**Discussion by marcelv-3** [erichelgeson/BlueSCSI-v1#86] [Ideas]
Build the source with an updated version of the sdfat library (and other libraries)
Matbe even more improved performance ?

**Comment by erichelgeson** [comment]
Thanks for the suggestion! I didnt notice any changes that would make a difference[0]. Have you tried it and did it have any improvements? 

[0] https://github.com/greiman/SdFat/releases

**Comment by marcelv-3** [comment]
have tested with sdfat 2.1.2
no change in read or write speed

external LCIII
write 767 kBytes/Sec
read 1258 kBytes/Sec

could be less variation in Typical rate speed with 2.1.2 if I run Time Drive multiple times 
but this requires more testing
